### ISSUE TITLE
Link to orphaned recipe 'Explain Code Structure'

### ISSUE THREAD

**Issue by bielsnohr** [ResearchCodeReviewCommunity/dev-review#62]
As mentioned in #61, we do actually have an 'orphaned' recipe called [Explain code structure](https://github.com/ResearchCodeReviewCommunity/dev-review/blob/main/site/content/recipes/explain_code_structure.md) to which we currently make no reference to anywhere else on the site!!! I think it is likely this will best fit as part of the 'Meet and Agree on Objectives' section, and should be handled in the 'startup meeting' mentioned there.

_Originally posted by @bielsnohr in https://github.com/ResearchCodeReviewCommunity/dev-review/issues/61#issuecomment-1084587395_

**Comment by tlestang** [comment]
Good catch, This overlaps with the first section of the "Perform code review" step: _Code author communicates code and context_.

Could we move the part where the code review is chosen to the initial meeting? If the code author is going to describe the code structure to the reviewer, it makes sense that this description is centered around the code under review, And if author is unsure about what code to review, they can discuss it with the reviewer at this stage.

**Comment by bielsnohr** [comment]
Yes, I think I tend to agree that the selection of the code segment should probably happen earlier in the process than we currently have it. I think it is very likely that code authors have a pretty good idea which part of their code they want reviewed prior to seeking the review.

Might be a bit late to try and get this in as a question at the CW22 workshop?
